charts/t8s-cluster/templates/management-cluster/clusterClass/openStackMachineTemplates/openStackMachineTemplates.yaml

```diff
@@ -1,9 +1,10 @@
 {{- range $name := list "compute-plane" "control-plane" }}
-{{- if false }}
+  {{- if or (ne $name "control-plane") (not $.Values.controlPlane.hosted) }}
+    {{- if false }}
```

Comment:
I guess you can remove everything between this if statement or do you use still need it for testing purposes?

Reply:
This is not for testing, this is for IntelliJ, as with every other CR we're using with dynamic apiVersion